prow/cmd/gcsupload/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 		logrus.WithError(err).Fatal("Could not resolve job spec")
 	}
 
+	logrus.Infof("gcsupload called with: %#+v", o)
```

Comment:
Can you `logrus.WithField("options": o).Debug("Called with options.")`?

Reply:
> Can you `logrus.WithField("options": o).Debug("Called with options.")`?

Shifted to `WithField` and moved into `Run` so I can say something more interesting than "Called with options" in b33a38c1d -> ce1c7a214.